aws_lambda_fsm/aws.py

```diff
@@ -445,7 +450,8 @@ def _get_redis_connection(cache_arn, entry):
 
             if host and port:
                 import redis
-                connection = redis.StrictRedis(host=host, port=port, db=0, ssl=ssl, password=password)
+                connection = redis.StrictRedis(host=host, port=port, db=0, ssl=ssl, password=password,
+                                               decode_responses=True)
```

Comment:
i'm going to guess this breaks currently cached stuff

Reply:
Is this just an observation?

Reply:
no. unencoded stuff is stored right now. decoding on get with have errors.

Reply:
For some context, the problem I was trying to solve here surfaces in the functional tests.
As noted [here](https://github.com/andymccurdy/redis-py#getting-started), this Redis library returns as bytes in Python 3. That causes a `TypeError` here:

```python
def _deserialize_lease_value(value):
    return map(int, value.split(':'))
```

Changing it to split on `b':'` causes errors with the Memcache smoke tests, because it *does* decode to a Python 3 (unicode) string.

As far as I can tell, this should work fine — if values are stored as Python 2 (ASCII) strings, that's equivalent to a bytestring, and decodable as UTF-8. Note that tests pass with minimal changes.